Patches/Simply More Melee/SMM_MeleeWeapons_CE.xml

```diff
@@ -0,0 +1,1476 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Simply More Melee</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+		<success>Always</success>
+			<operations>
+
+				<!-- SMM_Bardiche -->			
+
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="SMM_Bardiche"]/tools</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>handle</label>
+								<capacities>
+									<li>Blunt</li>
+								</capacities>
+								<power>8</power>
+								<cooldownTime>1.27</cooldownTime>
+								<armorPenetrationBlunt>2.925</armorPenetrationBlunt>
+								<chanceFactor>0.15</chanceFactor>
+							</li>
```

Comment:
You should include `linkedBodyPartsGroup` in each tool. This allows the injury description to read as `bardiche handle` instead of just `bardiche`, for example.

For a full list of values you can use, check the files at these locations:
> CombatExtended\Defs\Bodies\BodyPartGroups.xml 
> Core\Defs\Bodies\BodyPartGroups.xml

Reply:
Done